ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -541,7 +547,38 @@ const ControlledTierEraModule = (props: {profile: Profile, eligible: boolean, sp
     </FlexRow>;
   };
 
-  return isEnabledInEnvironment ? <Module data-test-id={`module-${AccessModule.ERACOMMONS}`} /> : null;
+  return isEnabledInEnvironment ? <Module data-test-id={`module-${moduleName}`} /> : null;
+};
+
+const ControlledTierFakeTrainingModule =
```

Comment:
i wonder if a different name makes sense here, which explains what it does better.

"CTTrainingSelfBypass" ?  something like that?